nservicebus/rabbitmq/cluster.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Using a RabbitMQ cluster 
+summary: Guidelines for using RabbitMQ in a cluster
+---
+
+The [RabbitMQ Clustering Guide](https://www.rabbitmq.com/clustering.html) recommends that you do not bake in the hostnames or IP addresses of the machines hosting nodes in a RabbitMQ cluster in your client applications and instead use an approach such DNS or load balancing. We make the same recommendation when using the NServiceBus RabbitMQ transport. For more details see the guide.
+
+If each of your clients is deployed to a machine which is running at least one RabbitMQ node in the cluster, another approach you may consider is to configure your client to only connect to the local node, e.g. using `localhost` as the hostname. Note that the RabbitMQ nodes must still be clustered, otherwise clients on each machine will only have access to the messages which originated locally.
+
```

Comment:
Ok, make sense

On Tue, Jun 9, 2015 at 3:34 PM, Adam Ralph <notifications@github.com> wrote:

> In nservicebus/rabbitmq/cluster.md
> <https://github.com/Particular/docs.particular.net/pull/652#discussion_r32012382>
> :
>
> > @@ -0,0 +1,10 @@
> > +---
> > +title: Using a RabbitMQ cluster
> > +summary: Guidelines for using RabbitMQ in a cluster
> > +---
> > +
> > +The [RabbitMQ Clustering Guide](https://www.rabbitmq.com/clustering.html) recommends that you do not bake in the hostnames or IP addresses of the machines hosting nodes in a RabbitMQ cluster in your client applications and instead use an approach such DNS or load balancing. We make the same recommendation when using the NServiceBus RabbitMQ transport. For more details see the guide.
> > +
> > +If each of your clients is deployed to a machine which is running at least one RabbitMQ node in the cluster, another approach you may consider is to configure your client to only connect to the local node, e.g. using `localhost` as the hostname. Note that the RabbitMQ nodes must still be clustered, otherwise clients on each machine will only have access to the messages which originated locally.
> > +
>
> Federation and shovel are very different to clustering so they are out of
> scope for this page.
>
> We *could* have a separate page for RabbitMQ distribution in general
> (clustering, federation and shovel) but I think this is probably out of
> scope for our docs. The important thing from a client perspective is
> connection to a logical broker which may or may not be clustered.
> Federation and shovel distribution are implemented at a lower level
> (exchange and queue) and are specifically configured server side, typically
> for distribution over a WAN, so I don't think this is something that will
> ever be a client concern. However, I've no direct experience of either, so
> please correct me if I'm wrong.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Particular/docs.particular.net/pull/652/files#r32012382>
> .
>